packages/@react-spectrum/slider/docs/RangeSlider.mdx

```diff
@@ -0,0 +1,152 @@
+```
+
+---
+category: Forms
+---
+
+# Slider - RangeSlider
+
+<p>{docs.exports.RangeSlider.description}</p>
+
+<HeaderInfo
+packageData={packageData}
+componentNames={['RangeSlider']}
+sourceData={[{type: 'Spectrum', url: 'https://spectrum.adobe.com/page/slider/'}]} />
+
+## Example
+
+```tsx example
+<RangeSlider label="Range" defaultValue={{start: 12, end: 36}} />
+```
+
+## Value
+
+RangeSliders are controlled with the `value` and `defaultValue` props.
+This value consists of two parts, start and end, both must fall between
+the RangeSlider's minimum and maximum values, which default to 0 and 100 respectively.
+
+```tsx example
+function Example() {
+  let [value, setValue] = React.useState({start: 25, end: 75});
+  return (
+    <Flex gap="size-150">
+      <RangeSlider
+        label="Range (Uncontrolled)"
+        defaultValue={{start: 25, end: 75}} />
+      <RangeSlider
+        label="Range (Controlled)"
+        value={value}
+        onChange={setValue} />
+    </Flex>
+  );
+}
+```
+
+Alternatively, a different scale can be used by setting the `minValue` and `maxValue` props.
+
+```tsx example
+<RangeSlider
+  label="Range"
+  minValue={50}
+  maxValue={150}
+  defaultValue={{start: 75, end: 100}} />
+```
+
+The slider value can be formatted by using the `formatOptions` prop.
+`formatOptions` is compatible with the option parameter of [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat) and is applied based on the current locale.
+
+```tsx example
+<RangeSlider
+  label="Price Range"
+  formatOptions={{style: 'currency', currency: 'JPY'}}
+  defaultValue={{start: 75, end: 100}} />
+```
+
+## Labeling
+Value labels are shown above the Slider by default, but they can also be moved to the side or hidden as needed.
+The label text should be in sentence case.
+
+```tsx example
+<Flex direction="column" maxWidth="size-3000" gap="size-300">
+  <RangeSlider label="Jeans Price Range" formatOptions={{style: 'currency', currency: 'USD'}} defaultValue={{start: 75, end: 100}} />
+  <RangeSlider label="Shoes Price Range" formatOptions={{style: 'currency', currency: 'USD'}} labelPosition="side" defaultValue={{start: 50, end: 100}} />
+  <RangeSlider label="Hats Price Range" formatOptions={{style: 'currency', currency: 'USD'}} showValueLabel={false} defaultValue={{start: 15, end: 30}} />
+</Flex>
+```
+
+By default, the value label is formatted as a plain number. This can be customized using the following props.
+
+* `showValueLabel`: Allows you to display or hide the value label.
+* `formatOptions`: Allows you to customize the format of the value.
+* `valueLabel`: Allows you to customize the value label to any string.
+
+```tsx example
+function Example() {
+  let [meters, setMeters] = React.useState({start: 15, end: 60});
+  return (
+    <Flex direction="column" maxWidth="size-3000" gap="size-300">
+      <RangeSlider
+        label="showValueLabel"
+        showValueLabel={false}
```

Comment:
bit odd that you have this labeled as "showValueLabel" but have `showValueLabel={false}` 
Perhaps
```suggestion
        label="No value label"
        showValueLabel={false}
```
or
```suggestion
        label="Level range"
        showValueLabel={false}
```
:P